src/Cake.NuGet/Install/NuGetSourceRepositoryProvider.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Cake.Core.Packaging;
+using NuGet.Configuration;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
+
+namespace Cake.NuGet.Install
+{
+    internal sealed class NuGetSourceRepositoryProvider : ISourceRepositoryProvider
+    {
+        private static readonly string[] _defaultSources =
+        {
+            "https://api.nuget.org/v3/index.json"
```

Comment:
@mholo65 should this come from the `cake.config` file?

Reply:
Yes, I think it should.

Reply:
Yes it should!